scripts/metalk8s

```
@@ -0,0 +1,39 @@
+#!/bin/bash
+# set -e
+set -u
+set -o pipefail
+
+SCRIPT_PATH="$(readlink -f "${BASH_SOURCE[0]}")"
+SCRIPTS_DIR="$(cd "$(dirname "${SCRIPT_PATH}")" && pwd)"
```

Hmm why `cd` + `pwd` ? `dirname "$SCRIPT_PATH"` is not enough ?

---

I think the rationale was "if `dirname ...` is not an actual directory, then it will fail early"

---

But I can remove it if you'd like

---

Hmm maybe simpler/cleaner to do `test -d` for this.
But I don't see a case where a `dirname` of something who come from a `readlink -f` can give a non-directory file :confused: